src/ExpressionEvaluator/Core/Source/ExpressionCompiler/ExpressionCompiler.cs

```diff
@@ -23,6 +22,7 @@ namespace Microsoft.CodeAnalysis.ExpressionEvaluator
     public abstract class ExpressionCompiler :
         IDkmClrExpressionCompiler,
         IDkmClrExpressionCompilerCallback,
+        IDkmMetaDataPointerInvalidatedNotification,
         IDkmModuleModifiedNotification,
```

Comment:
>IDkmModuleModifiedNotification [](http://example.com/codeflow?start=8&length=30)

You shouldn't need to implement this anymore (unless this is just a temporary thing to keep tests happy until you are running against a new debugger)

Reply:
I think I'd prefer keeping the implementation of `IDkmModuleModifiedNotification`, for robustness, assuming it doesn't conflict with `IDkmMetaDataPointerInvalidatedNotification`.

Reply:
They will not conflict - just a bit of a perf overhead since now Rosyln will be notified twice anytime, we modify an assembly.